docs/arguments.md

```diff
@@ -234,9 +234,9 @@ It also talks to a KVStore and has it's own copies of the same flags used by the
 
 - `-ingester.normalise-tokens`
 
-   Write out "normalised" tokens to the ring.  Normalised tokens consume less memory to encode and decode; as the ring is unmarshalled regularly, this significantly reduces memory usage of anything that watches the ring.
+   Deprecated. New ingesters always write "normalised" tokens to the ring. Normalised tokens consume less memory to encode and decode; as the ring is unmarshalled regularly, this significantly reduces memory usage of anything that watches the ring.
```

Comment:
This change should be also clearly stated out in the changelog.

Reply:
I don't want to update changelog yet, as 0.4.0 is going to be released soon (13th nov), and then I'll need to change it anyway. But you're right, this needs to go to changelog.

Reply:
Added changelog entry now.